charts/everest/README.md

```diff
@@ -12,51 +12,82 @@ but if you do so, please be aware of the risks.
+helm repo update
 ```
 
-> Note: we currently do not support deploying Everest in a namespace other than `everest-system`.
+### 2. Install Everest
+```sh
+helm repo add percona https://percona.github.io/percona-helm-charts/
+helm install everest-core percona/everest \
+    --namespace everest-system
+    --create-namespace
+```
 
-This command may take a few minutes to complete. Once done, you can retrieve the admin credentials using the following command:
+Notes:
+* This command deploys the Everest components in the `everest-system` namespace. Currently, we do not support specifying a different namespace for Everest.
+* Additionally, it also deploys a new namespace called `everest` for your databases and the database operators.
+* If you prefer to manage your database namespace separately, you may set `dbNamespace.enabled=false`.
+* You may override the name of the database namespace using the `dbNamespace.namespaceOverride` parameter.
+* By default, all database operators are installed in your database namespace. You may override this by specifying one or more of the following: [`dbNamespace.pxc=false`, `dbNamespace.pg=false`, `dbNamespace.psmdb=false`].
+* We currently do not support installation without the use of chart hooks. I.e, the use of `--no-hooks` is not supported during installation.
 
+### 3. Retrieve the admin password
+
+Once the installation is complete, you may retrieve the admin credentials using the following command:
 ```sh
 kubectl get secret everest-accounts -n everest-system -o jsonpath='{.data.users\.yaml}' | base64 --decode  | yq '.admin.passwordHash'
 ```
 
-### Deploy your database namespace components
+You may open the Everest UI by port-forwarding the service to your local machine:
+
+```sh
+kubectl port-forward svc/everest -n everest-system 8080:8080
+```
+
+Notes:
+* The default username to login to the Everest UI is `admin`.
+* You may specify a different default admin password using `server.initialAdminPassword` parameter during installation.
+* The default admin password is stored in plain text. It is highly recommended to update the password using `everestctl` to ensure that the passwords are hashed.
+
+### 4. Deploy additional database namespaces
 
-Once Everest is running, we need to create a namespace for your databases and provision the necessary operators.
+After Everest is successfully running, you may create additional database namespaces using the `everest-db-namespace` Helm chart.
+If you had set `dbNamespaces.enabled=false` in the previous step, you may deploy a database namespaces using the following command:
 
 ```sh
-cat <<EOF | kubectl create -f -
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: everest
-  labels:
-    app.kubernetes.io/managed-by: everest
-EOF
-helm install everest percona/everest-db-namespace --namespace everest
+helm install everest \
+    percona/everest-db-namespace \
+    --namespace everest
```

Comment:
Why doesn't this need the `--create-namespace`?

Reply:
I just saw that you have the `charts/everest/charts/everest-db-namespace/templates/namespace.yaml`. Is that why we don't need the `--create-namespace`?

Since you have the hook that labels the namespace couldn't we leverage the `--create-namespace` feature of helm?

Reply:
It needs the flag, I missed it in the docs. Thanks for catching it :)

 > I just saw that you have the charts/everest/charts/everest-db-namespace/templates/namespace.yaml. Is that why we don't need the --create-namespace?

No, that template is rendered only when you deploy the sub-chart with the main chart, since Helm will not create that namespace for you otherwise